ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productlist/v1/productlist/clientlibs/dist/js/navhelper.js

```diff
@@ -0,0 +1,27 @@
+let getCookie = function (cname) {
```

Comment:
Why is that needed? Do we store the page state in a cookie?

Reply:
earlier it was designed like that, but later we discussed and agreed to keep page movements tracked via CGI params in query String, these can be used later, or can be removed - you suggest we remove this?

Reply:
I opt for removing it. We can add it later if needed.

Reply:
+1 for removing it. This is not a requirement for the ticket. 

@deepkamal "earlier it was designed like that"? This is no earlier version of this? What are you referring to?

Reply:
I'll remove it.
Pardon my English , by "earlier it was designed like that" I mean "earlier I started with using cookies, later switched to query string".

Reply:
nvhelper.js removed, page switching arithmetic now included in Model methods